mono/metadata/object-internals.h

```diff
@@ -38,6 +38,15 @@
 	} 								\
 } while (0)
 
+#define MONO_CHECK_ARG_NULL_NAMED(arg, argname, retval) do {	\
+	if (G_UNLIKELY (!(arg)))				\
+	{							\
+		ERROR_DECL (error);				\
+		mono_error_set_argument_null (error, (argname), "");	\
+		mono_error_set_pending_exception (error);	\
+		return retval;					\
```

Comment:
Just a nit: You can remove `ERROR_DECL` and `mono_error_set_pending_exception`, in the uses so far. Perhaps not in future uses.